src/metadataGeneration/tsoa.ts

```diff
@@ -39,6 +39,11 @@ export namespace Tsoa {
     validators: Validators;
   }
 
+  export interface ArrayParameter extends Parameter {
```

Comment:
Why introduce this?

Reply:
To avoid defining the `collectionFormat` in `Parameter`, because it is specific to array parameters. There was already an `ArrayType` for array-specific types so I thought it would be fair to also have an `ArrayParameter`.